docs/books/common/attributes.adoc

```diff
@@ -44,7 +44,7 @@ under the License
 :RouterName: Dispatch Router
 :RouterSchemaDir: ../../build/doc/book
 :RouterConfigFile: /etc/qpid-dispatch/qdrouterd.conf
-:DispatchRouterVersion: 1.19.0-SNAPSHOT
+:DispatchRouterVersion: 2.0.0-SNAPSHOT
```

Comment:
Should this be called SkupperRouterVersion ?

Reply:
I'm never sure what is and what is not in scope of each of these 'dispatch to skupper' transition PRs.

Reply:
@ganeshmurthy No - I simply ran the version update process I usually do for releasing qpid dispatch.   There's about a gazillion (give or take a few bazillion) uses of the term "dispatch" throughout the code.  This patch does nothing to change that.  All this patch does is change the version number.  And changes to documentation are way way way way way (did I say 'way') beyond the scope of this patch.

Reply:
Perhaps https://github.com/skupperproject/skupper-router/pull/141/files could address this?